gems/pending/appliance_console/database_configuration.rb

```diff
@@ -64,6 +64,7 @@ def activate
 
     def post_activation
       ServiceGroup.new.restart_services
+      evm_service = LinuxAdmin::Service.new("evmserverd").enable.start
```

Comment:
@carbonin Why are we setting the local variable `evm_service` just as method is about to exit?  Or is it not a local variable?